packages/wrangler/src/index.ts

```diff
@@ -721,6 +722,9 @@ export async function main(argv: string[]): Promise<void> {
 			// The workaround is to re-run the parsing with an additional `--help` flag, which will result in the correct help message being displayed.
 			// The `wrangler` object is "frozen"; we cannot reuse that with different args, so we must create a new CLI parser to generate the help message.
 			await createCLIParser([...argv, "--help"]).parse();
+		} else if (isAuthenticationError(e)) {
+			logger.log(formatMessage(e));
+			await whoami();
```

Comment:
Should this return or throw?

Reply:
No, it's thrown at the end of this function